packages/amplify_datastore/example/lib/main.dart

```diff
@@ -135,19 +136,19 @@ class _MyAppState extends State<MyApp> {
   }
 
   void listenToHub() {
-    AmplifyDataStore.events.listenToDataStore((msg) {
-      if (msg["eventName"] == "ready") {
-        runQueries();
-      }
-      print(msg);
-    });
     setState(() {
+      hubSubscription = Amplify.Hub.listen([HubChannel.DataStore], (msg) {
+        if (msg["eventName"] == "ready") {
```

Comment:
shouldn't this be changed to msg.eventName for correctness?

Reply:
Yup, done.